vignettes/epidist.Rmd

```diff
@@ -201,29 +201,31 @@ With our censored, truncated, and sampled data, we are now ready to try to recov
 
 # Fit the model and compare estimates {#fit}
 
-If we had access to the data `obs`, then it would be simple to estimate the delay distribution.
+If we had access to the complete and unaltered `obs`, it would be simple to estimate the delay distribution.
 However, with only access to `obs_cens_trunc_samp`, the delay distribution we observe is biased (Figure \@ref(fig:obs-est)) and we must use a statistical model.
 
-(ref:obs-est) The histogram of delays from `obs` matches closely the underlying distribution, whereas those from `obs_cens_trunc_samp` are systematically biased.
+(ref:obs-est) The histogram of delays from the complete retrospective data `obs_cens` match closely the underlying distribution, whereas those from `obs_cens_trunc_samp` are systematically biased. In this instance, while the complete, retrospective data are still subject to censoring, the extent of this issue is quite minor as compared with that from right truncation.
```

Comment:
I don't think we should be downplaying censoring like this IMO. 

The teaching point is that the real-time data doesn't look like the underlying distribution and nor does the fully observed data but usually to a lesser degree. The key point being that both need a model to map to the underlying distribution. 

We can then add another teaching point which is that if we knew the infection time (and stress we likely never will) then observed censored daily data (i.e your daily histogram) we would have no bias (just increased obs error from the censoring and sampling variation).

Reply:
One way to make this point (as I think @parksw3) is suggesting is to plot the mean and sd of the empirically observed data at different times vs the underlying true distribution. you could also add sampling variation to this if you wanted. This would then help you with some of your below points about "being close to the data"

Reply:
![image](https://github.com/epinowcast/epidist/assets/29818116/3095e80f-2ed9-4210-af32-72a1db8f9317)

Here's the image that this is a caption for. It looks to me that the main issue here is the trunaction rather than the censoring (here we also have the sampling). So yes, it's a lesser degree, which the text does say. Perhaps I'd prefer to write "relatively" minor rather than "quite" minor.

Reply:
My point here is we are still saying people should use a model in most cases

Reply:
> plot the mean and sd of the empirically observed data at different times vs the underlying true distribution. you could also add sampling variation to this if you wanted. 

I think this would add something to understanding. It might be quite a lot of code / complication for the getting started vignette, which I'd prefer to keep simple if possible. Happy to make another issue for this if you'd like.

Reply:
I am thinking of adding another sentence along the lines of "that said, we still think it's important to use a model to take into account censoring...". Is there a relevant little section or citation we could point to here for people who are interested? Could also just say in which situations it'd be more of an issue.

Reply:
@seabbs if there are any changes you'd like to make to get this over the line I'm happy for you go to into the branch.

Reply:
> I think this would add something to understanding. It might be quite a lot of code / complication for the getting started vignette, which I'd prefer to keep simple if possible. Happy to make another issue for this if you'd like.

Yes that makes sense. I am suggesting this vs trying to pack in more context to the above plot.

> s there a relevant little section or citation we could point to here for people who are interested? Could also just say in which situations it'd be more of an issue.

The long delay paper and Kelly's paper both cover this. 

> "that said, we still think it's important to use a model to take into account censoring.

Yes agree. If we want to make this clearer we can just shift to using a slightly shorter delay as the censoring bias is absolute

Reply:
I've rewritten this caption. If there are specific changes you would like me to make please write them down and I will do them.

Reply:
Unsure if I should resolve here as the above caption is outdated.

Reply:
Prior to resolving I think it would be useful to either respond to some of the above suggestions (i.e a shorter delay), move to issues, or adapt the current text in response to them